deps/rabbit/test/feature_flags_SUITE.erl

```diff
@@ -551,7 +547,9 @@ registry_concurrent_reloads(_Config) ->
     MRef = erlang:monitor(process, Spammer),
     unlink(Spammer),
     exit(Spammer, kill),
-    receive {'DOWN', MRef, process, Spammer, _} -> ok end.
+    receive {'DOWN', MRef, process, Spammer, _} -> ok end,
+    %% All feature flags appeared.
+    ?assertEqual(FinalFFList, ?list_ff(all)).
```

Comment:
This assertion fails on main but it's because of a bug in the test helpers (see 2e4e321c5a8eb80c3397053b5703f1d4ed495ccd) rather than a bug in the feature flag system